integration-templates/sharepoint-online/nango.yaml

```diff
@@ -8,31 +8,38 @@ integrations:
                 scopes:
                     - Sites.ReadWrite.All
                     - offline_access
-                output: SharePointSiteId
+                output: SharePointMetadata
         syncs:
             file-sync:
                 description: |
                     This sync will be used to sync file metadata from SharePoint site based on the ones the user has picked.
-                endpoint: /sharepoint/file-sync
+                endpoint: GET /sharepoint/file-sync
                 sync_type: incremental
-                track_deletes: true
                 runs: every 1 hour
                 scopes:
                     - Sites.ReadWrite.All
                     - offline_access
-                auto_start: true
+                input: SharePointMetadata
+                auto_start: false
                 output: FileMetadata
```

Comment:
```suggestion
                output: FileMetadata
                version: 1.0.0
```

Reply:
We can talk about this versioning piece during our next sync

Reply:
@khaliqgant, sure, let's discuss the versioning piece during our next sync. I think it might be related to the `version@undefined` error I was encountering earlier during `nango deploy`.